apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/interceptor/enhance/v2/ClassEnhancePluginDefineV2.java

```diff
@@ -56,15 +57,22 @@
  */
 public abstract class ClassEnhancePluginDefineV2 extends AbstractClassEnhancePluginDefine {
 
+    private ConstructorInterceptPoint[] constructorInterceptPoints;
+    private InstanceMethodsInterceptV2Point[] instanceMethodsInterceptV2Points;
+    private StaticMethodsInterceptV2Point[] staticMethodsInterceptV2Points;
```

Comment:
I think these should stay inside method scope? The classes were stateless before

Reply:
Cache the `XxxInterceptPoint`  instances which used by  `DelegateNamingResolver#resolve(Object interceptPoint)`  to generate fixed field name on re-transform class